services/base-service/spa/controllers/static.js

```diff
@@ -42,6 +44,21 @@ module.exports = async (app) => {
         if( !user.roles ) user.roles = [];
         if( user.roles.includes('admin') ) user.admin = true;
         user.loggedIn = true;
+        user.expertId = 'expert/'+ md5(user.preferred_username+'@ucdavis.edu');
+
```

Comment:
async client_get(id,options) {
    // console.log(`FinEsNestedModel.client_get(${id}) on ${this.readIndexAlias}`);
    const result = await this.client.get(
      {
        ...{
          index: 'experts-read'
          id: md5(user.preferred_name+"@ucdvis.edu"
        },
      }
    )
    return result._source;
    }

Reply:
@qjhart 
I've updated the code to use that pattern. The `esClient.get()` throws an exception when nothing is found, so I wrapped it in a try/catch. Seems to function as expected.